### PR TITLE
JCache samples added

### PR DESCRIPTION
**Pull request by aq47**

### CODE REVIEW

File: JCache-Examples/JCacheCDI/pom.xml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        <endorsed.dir>${project.build.directory}/endorsed</endorsed.dir>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+    
+    <dependencies>
+        <dependency>
+            <groupId>fish.payara.appserver</groupId>
+            <artifactId>jsr107-repackaged</artifactId>
+            <version>4.1.154-SNAPSHOT</version>
+            <type>jar</type>
```
Comment (smillidge) [inline JCache-Examples/JCacheCDI/pom.xml]: Can you set the scope as provided

File: JCache-Examples/JCacheInjection/pom.xml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+    
+    <dependencies>
+        <dependency>
+            <groupId>fish.payara.appserver</groupId>
+            <artifactId>jsr107-repackaged</artifactId>
+            <version>4.1.154-SNAPSHOT</version>
+            <type>jar</type>
+        </dependency>
```
Comment (smillidge) [inline JCache-Examples/JCacheInjection/pom.xml]: Can  you set the scope to provided

**Comment by smillidge** [comment]
Can you add README.md file to the example and ensure the JCacheExamples is added as a module to the parent pom

**Comment by aq47** [comment]
Have a look now Steve, thanks

**Pull request merged**